CMakeExt/CompilerFlags.cmake

```diff
@@ -36,7 +36,7 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES ".*Clang")
   # using Clang
   set (CXX_STD_FLAG "--std=c++11"
        CACHE STRING "C++ compiler std flag")
-  set (CXX_OMP_FLAG "-fopenmp")
+  set (CXX_OMP_FLAG "-fopenmp=libomp")
```

Comment:
This is not clang compatible, at least with version 3.4.2 (tags/RELEASE_34/dot2-final)
```
clang: error: unknown argument: '-fopenmp=libomp'
```